ibu-imager/cmd/create.go

```diff
@@ -70,20 +71,16 @@ func init() {
 	createCmd.Flags().StringVarP(&authFile, "authfile", "a", imageRegistryAuthFile, "The path to the authentication file of the container registry.")
 	createCmd.Flags().StringVarP(&containerRegistry, "image", "i", "", "The full image name with the container registry to push the OCI image.")
 	createCmd.Flags().StringVarP(&recertContainerImage, "recert-image", "e", defaultRecertImage, "The full image name for the recert container tool.")
+
+	createCmd.MarkFlagRequired("image")
+	createCmd.MarkFlagRequired("recert-image")
```

Comment:
If we have a default recert image value, we don't need to mark the flag as "required", making it transparent to existing workflows